docs/_articles/en/codebuilder/cb-diffs.md

```diff
@@ -0,0 +1,24 @@
+---
+title: What's Different
+lang: en
+---
+## What’s Different in Code Builder
+
+Code Builder offers the same rich tooling and unleashes the power of Salesforce VS Code extensions  as the desktop version. No matter whether you are using VS Code on desktop or Code Builder from a browser, the Salesforce extensions you access are the same. While the extensions generally function the same way regardless of how they are being accessed, there are some nuances to be aware of:
+
+* Code Builder installed as a managed package, and your Code Builder environment comes pre-configured for you.
+* Working in Code Builder doesn’t require you to install VS Code or Salesforce Extensions or Salesforce CLI. 
+* Unlike VS Code for the desktop, you cannot add any new extensions to your Code Builder development environment.
+* Because Code Builder is inherently lightweight and optimized to run in the cloud, access to certain functionalities is limited:    
+    * Functions development is available for Javascript and Typescript Functions only.
+    * LWC Local Development (beta) is not available in Code Builder. Deploy LWC components to your org to view them.
```

Comment:
should you say "preview" somewhere. (second sentence) You can't preview LWC components locally but must deploy them to your org to view them. (something like that)